code/game/objects/items/devices/doorCharge.dm

```diff
@@ -1,5 +1,5 @@
 /obj/item/doorCharge
-	name = "airlock charge"
+	name = "Airlock Charge"
```

Comment:
why are you changing the name? This shouldn't be capital since it isn't a proper noun and the game handles start of sentance capitilization for you

Reply:
Oh didn't know that